src/index.html

```diff
@@ -12,14 +12,10 @@ <h1>Leaderboard</h1>
             <div>
                 <h2>Recent scores</h2>
                 <div class="button">
-                    <button type="button">Refresh</button>
+                    <button type="button" id="refresh-scores">Refresh</button>
                 </div>
             </div>
```

Comment:
- [Optional] The resent score list is empty when the page loads, It only renders the scores once the user hit refresh. If possible it would be great to have an auto refresh. `when a new score is added refresh the list of scores`. Otherwise, it could be great to load the list of scores when the page loads (without waiting for the user to refresh manually).

### The list of scores is empty on reload.
![Capture](https://user-images.githubusercontent.com/69077061/204103127-6f4d8d76-97f2-4d08-9e2f-68a224372c2c.PNG)